src/__stories__/navigation-bar-story.tsx

```diff
@@ -15,13 +15,14 @@ export default {
     },
 };
 
-type Args = {isInverse: boolean; withBorder: boolean};
+type Args = {isInverse: boolean; withBorder: boolean; withDivider: boolean};
 
-export const Default: StoryComponent<Args> = ({isInverse, withBorder}) => {
+export const Default: StoryComponent<Args> = ({isInverse, withBorder, withDivider}) => {
     const {isDesktopOrBigger} = useScreenSize();
     return (
         <NavigationBar
             withBorder={withBorder}
+            withDivider={withDivider}
```

Comment:
repeated? withBorder and withDivider

Reply:
Removed withDivider and made withBorder optional in the places where it was forced to true